packages/account/src/Components/poi/status/unsupported/card-details/uploader.tsx

```diff
@@ -124,36 +143,30 @@ const Uploader = ({ data, value, is_full, onChange, has_frame }) => {
                         filename_limit={32}
                         hover_message={localize('Drop files here..')}
                         max_size={max_document_size}
-                        message={open => <Message open={open} data={data} />}
+                        message={(open: FormikValues) => <Message open={open} data={data} />}
                         preview_single={
                             image && (
                                 <Preview
                                     data={data}
                                     value={image}
                                     has_frame={has_frame}
                                     setFieldValue={setFieldValue}
-                                    handleChange={handleChange}
+                                    handleChange={() => handleChange()}
                                 />
                             )
                         }
                         multiple={false}
-                        onDropAccepted={files => handleAccept(files, setFieldValue)}
-                        onDropRejected={files => handleReject(files, setFieldValue)}
+                        onDropAccepted={(files: object[]) => handleAccept(files, setFieldValue)}
+                        onDropRejected={(files: Array<object>) => handleReject(files, setFieldValue)}
                         validation_error_message={value?.errors?.length ? ValidationErrorMessage : null}
                         noClick
                         value={image ? [image] : []}
+                        className=''
```

Comment:
🤔

Reply:
sent the card back to blocked since FileDropZone is blocking the component.

Reply:
@suisin-deriv could you please convert the PR to draft as well?